src/compensations/components/benefitsByLocation/BenefitsByLocation.tsx

```diff
@@ -0,0 +1,29 @@
+import { Benefit } from "studio/lib/payloads/compensations";
+import Text from "src/components/text/Text";
+import { RichText } from "src/components/richText/RichText";
+import styles from "./benefitsByLocation.module.css";
+
+interface BenefitsByLocationProps {
+  locationId: string;
+  benefits?: Benefit[] | any;
```

Comment:
`benefits` should have the same type as the actual input, i.e. `BenefitsByLocation[]` (should rename to `BenefitsByLocationPage` to not conflict with the React component)

Reply:
I forgot that i wrote any to test it😬 Will fix it now!